app/_includes/md/enterprise/cassandra-deprecation.md

```diff
@@ -1,7 +1,16 @@
+{% if include.length == "short" %}
+
+We don't recommend using Cassandra with {{site.base_gateway}},
+as support for Cassandra is [deprecated and planned to be removed](/gateway/{{include.kong_version}}/reference/configuration/#cassandra-settings).
+
+{% else %}
+
 {:.warning .no-icon}
-> **Deprecation warning:** Cassandra as a backend database for Kong Gateway
-is deprecated. This means the feature will eventually be removed.
+> **Deprecation warning:** Cassandra as a backend database for {{site.base_gateway}}
+is deprecated. Support for Cassandra will eventually be removed.
```

Comment:
```suggestion
is deprecated. Support for Cassandra will be removed.
```

Reply:
"Support for Cassandra will be removed in a future release"? If we leave it as "Support for Cassandra will be removed.", that could be implying that it'll be removed at any moment.